pulp_smash/tests/rpm/api_v2/test_orphan_remove.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+"""Test the removal of rpm orphans.
+
+This module assumes that the tests in
+:mod:`pulp_smash.tests.platform.api_v2.test_repository` and
++`pulp_smash.tests.platform.api_v2.test_sync_publish` hold true.
+"""
+
+from __future__ import unicode_literals
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin  # pylint:disable=C0411,E0401
+
+from pulp_smash import api, utils
+from pulp_smash.constants import (
+    ORPHANS_PATH,
+    REPOSITORY_PATH,
+    RPM_FEED_URL,
+)
+
+
+def _gen_repo():
```

Comment:
FYI, `pulp_smash.tests.rpm.api_v2.utils` has a function `gen_repo`. The above can become:

```python
from pulp_smash import api, utils
from pulp_smash.constants import ORPHANS_PATH, REPOSITORY_PATH, RPM_FEED_URL
from pulp_smash.tests.rpm.api_v2.utils import gen_repo
```